### DISCUSSION TITLE
[QUESTION] Deselect when options list is changed

### DISCUSSION THREAD

**Discussion by gbspecapedra** [MobileReality/react-native-select-pro#96] [Q&A] [answered]
Hello folks, at this moment I'm trying to deselect an option that does not exist in the list options depending on the selection of another select field.

For example, in a select field for a day and another for a month, the select day has 31 days but when I select FEB in the month select, day 31 doesn't exist in the options list in the first select but is not cleaned from the value selected.

Any ideas how to fix that?

**Comment by irekrog** [comment, marked as answer]
Hi @wdgisele 

You can try to use [onSelect](https://mobilereality.github.io/react-native-select-pro/docs/api/callbacks#onselect) and [clear()](https://mobilereality.github.io/react-native-select-pro/docs/api/refs#clear). 

Look at the example: https://snack.expo.dev/@irekrog/smelly-beef-jerky-2
